### PR TITLE
Add route to ingest supplier details to application flow

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/main/views/suppliers.py [hunk 1]
```diff
@@ -110,6 +110,43 @@ def edit_supplier(supplier=None, errors=None):
...
+
+    try:
+        supplier = data_api_client.get_supplier(
+            current_user.supplier_code
+        )['supplier']
+    except APIError as e:
+        current_app.logger.error(e)
+        abort(e.status_code)
+
+    if feature.is_active('SELLER_REGISTRATION'):
```
Comment (matt-sm) [inline app/main/views/suppliers.py]: I wouldnt use the same feature flag

Comment (maxious) [inline app/main/views/suppliers.py]: Okay, added a different one.

File: app/main/views/suppliers.py [hunk 2]
```diff
@@ -110,6 +110,43 @@ def edit_supplier(supplier=None, errors=None):
...
+            current_user.supplier_code
+        )['supplier']
+    except APIError as e:
+        current_app.logger.error(e)
+        abort(e.status_code)
+
+    if feature.is_active('SELLER_UPDATE'):
+        # add business/authorized representative contact details
+        if len(supplier['contacts']) > 0:
+            supplier['email'] = supplier['contacts'][0]['email']
```
Comment (djrobstep) [inline app/main/views/suppliers.py]: Possibly better as a try_catch to get the first contact? Is this get first contact robust to multiple contacts? How are we using the role column at present? Do we need to change this?

File: app/main/views/suppliers.py [hunk 3]
```diff
@@ -110,6 +110,43 @@ def edit_supplier(supplier=None, errors=None):
...
+            supplier['phone'] = supplier['contacts'][0]['phone']
+            supplier['representative'] = supplier['contacts'][0]['name']
+        if len(supplier['address']) > 0:
+            supplier['address']['address_line'] = supplier['address']['addressLine']
+            supplier['address']['postal_code'] = supplier['address']['postalCode']
```
Comment (djrobstep) [inline app/main/views/suppliers.py]: This props already exist in snake case on the address object returned for a supplier I thought?

**Pull request merged**